web/pgadmin/browser/__init__.py

```diff
@@ -683,7 +684,8 @@ def utils():
             qt_default_placeholder=QT_DEFAULT_PLACEHOLDER,
             vw_edt_default_placeholder=VW_EDT_DEFAULT_PLACEHOLDER,
             enable_psql=config.ENABLE_PSQL,
-            pgadmin_server_locale=default_locale
+            pgadmin_server_locale=default_locale,
+            is_nwjs='true' if current_app.PGADMIN_RUNTIME else 'false',
```

Comment:
We don't need this. We already have a JS function `getBrowser` for this. You can:
```
    let {name: browser} = getBrowser();
    if(browser == 'Nwjs') {
```